vendor/github.com/moby/sys/mountinfo/mountinfo_linux.go

```diff
@@ -1,4 +1,6 @@
-package mount // import "github.com/docker/docker/pkg/mount"
+// +build go1.13
```

Comment:
**Hello, I wonder if the build condition here has a special meaning?**

----
```
#37 0.675 Removing /build/bundles/
 #37 0.677 
 #37 0.684 ---> Making bundle: binary (in /build/bundles/binary)
 #37 0.821 Building: /build/bundles/binary-daemon/dockerd-0.0.0-20200404132736-92768b3296
 #37 0.823 GOOS="" GOARCH="" GOARM=""
 #37 33.99 # github.com/docker/docker/vendor/github.com/moby/sys/mountinfo
 #37 33.99 vendor/github.com/moby/sys/mountinfo/mountinfo.go:8:9: undefined: parseMountTable
 #37 33.99 vendor/github.com/moby/sys/mountinfo/mountinfo.go:16:9: undefined: parseInfoFile
 #37 ERROR: executor failed running [/bin/sh -c hack/make.sh binary]: runc did not terminate sucessfully
 ------
  > [build-binary 1/1] RUN --mount=type=cache,target=/root/.cache/go-build     --mount=type=bind,target=/go/src/github.com/docker/docker         hack/make.sh binary:
 ------
 failed to solve with frontend dockerfile.v0: failed to solve with frontend gateway.v0: rpc error: code = Unknown desc = failed to build LLB: executor failed running [/bin/sh -c hack/make.sh binary]: runc did not terminate sucessfully
 make[2]: *** [Makefile:188: binary] Error 1
 make[2]: Leaving directory '/builds/[MASKED]/docker-ce-packaging/docker-ce/components/engine'
 make[1]: *** [Makefile:73: static-engine] Error 2
 make[1]: Leaving directory '/builds/[MASKED]/docker-ce-packaging/static'
 make: *** [Makefile:35: static] Error 2
```

I used GitLab CI for nightly build, but the recent task failed, and the above error message was found in the log.
After checking, I found that it was caused by the build condition here. (the build job using golang 1.12)

Reply:
Yes, this package requires Go 1.12 or up

Reply:
Thanks for your reply.
Go 1.12 or up? Should we change the  build condition?

Reply:
oh, sorry, wanted to type 1.13, and mis-typed 😂

Reply:
I have updated my build job to use 1.13 and it works fine. 

By the way, I want to ask, what is the reason why it needs 1.13? I checked the code and did not see the features that must be supported in 1.13, I don’t know if I missed something. Thanks!